### DISCUSSION TITLE
RBAC

### DISCUSSION THREAD

**Discussion by turtle0x1** [turtle0x1/LxdMosaic#519] [Ideas]
So we added project restrictions, who cares;

We need to add RBAC, this is another huge amount of effort that needs to be defined; 

 - Who can schedule backups
 - Who can create instances
 - Who can create / edit each thing in the navbar
 

Maybe look to LXD for inspiration, either way huge amount of effort!

**Comment by turtle0x1** [comment]
When we add this must make sure to make all users admin, `0.11.0` was annoying enough for end users when they had to assign user to every host & project!

**Comment by aivanise** [comment]
To me it is important to restrict users in a way so that they can not (easily) use up the cluster, so start/stop or seeing what is going on within a certain conainer/project is ok, but there should be some kind of a budget how many resources (RAM, diskspace, CPU cores?) a user can 'own' and they should not be able to create more instances or extend existing ones if the budget is used up.

**Comment by turtle0x1** [comment]
Im on my 2nd implementation, unusual for me, yet i'm still not happy.

RBAC is related to more than LXD and im not going to close this based on my "naive" implementation.

I will migrate this to "discussions" though because "issues" are becoming a place for things i actually need to "fix" not "big picture" things.

Thanks
turtle0x1
